pkg/api/bytes_test.go

```diff
@@ -67,9 +67,9 @@ func TestBytes(t *testing.T) {
 	})
 
 	t.Run("not found", func(t *testing.T) {
-		jsonhttptest.ResponseDirect(t, client, http.MethodGet, resource+"/abcd", nil, http.StatusNotFound, jsonhttp.StatusResponse{
-			Message: "not found",
-			Code:    http.StatusNotFound,
+		jsonhttptest.ResponseDirect(t, client, http.MethodGet, resource+"/abcd", nil, http.StatusBadRequest, jsonhttp.StatusResponse{
+			Message: "invalid root chunk",
+			Code:    http.StatusBadRequest,
```

Comment:
This should be a valid request, just the chunk is not found. I think that 404 is correct to return.

Reply:
the inner error is `invalid reference length` which is due to some more strict checks that i am now doing further down the call stack. the resource is `/abcd` so i actually think that `bad request` is legible here. if you insist i'll change it. the message should not be 'invalid root chunk' tho. probably just `bad request`